data/sql_updates/create_omnibus_dates.sql

```diff
@@ -1,14 +1,36 @@
+drop materialized view if exists ofec_omnibus_dates_mv_tmp;
+
+-- pre-processing to get the right district-state combos for House members and
+-- the election doesn't exist if the status ID is not 1
+drop view if exists tmp_election_vw;
+create view tmp_election_vw as
+    select
+        *,
+        case
+            when office_sought = 'H' and election_district != ' ' then array_to_string(
+                array[
```

Comment:
With just two values, I think `election_state || '-' || election_district` would be easier to read.